admin/static/js/pages/prereg-admin-page.js

```diff
@@ -0,0 +1,254 @@
+'use strict';
+
+var $osf = require('js/osfHelpers');
+var ko = require('knockout');
+var $ = require('jquery');
+
+var drafts;
+
+ko.bindingHandlers.enterkey = {
+    init: function (element, valueAccessor, allBindings, viewModel) {
+        var callback = valueAccessor();
+        $(element).keypress(function (event) {
+            var keyCode = (event.which ? event.which : event.keyCode);
+            if (keyCode === 13) {
+                callback.call(viewModel);
+                return false;
+            }
+            return true;
+        });
+    }
+};
+
+/**
+* Assignee column.
+*
+* @param  {List} reviewers  The prereg reviewers who can be assigned
+*/
+var Assignee = function(reviewers) {
+    var self = this;
+    self.edit = ko.observable(false);
+
+    if (reviewers[0] !== 'none') {
+        reviewers.unshift('none'); 
+    }
+    
+    self.reviewers = ko.observableArray(reviewers);
+    self.assignee = ko.observable('none');
+};
+
+Assignee.prototype.editItem = function() {
+    var self = this;
+    self.assignee.edit(!self.editing);
+    self.editing(!self.editing);
+    
+};
+
+/**
+* Proof of publication column.
+*/
+var ProofOfPub = function() {
+    var self = this;
+    self.edit = ko.observable(false);
+    self.proofOfPub = ko.observable('Published Article Not Yet Submitted');
+    self.proofOfPubList = ko.observableArray(['Published Article Not Yet Submitted', 'Published Article Submitted', 'Published Article Under Review', 'Published Article Approved', 'Published Article Rejected']);
+};
+
+ProofOfPub.prototype.editItem = function() {
+    var self = this;
+    self.proofOfPub.edit(!self.editing);
+    self.editing(!self.editing);
+};
+
+/**
+* Payment sent column.
+*/
+var PaymentSent = function() {
```

Comment:
For some reason, in my test pre-reg, in the admin side my dropdown was defaulted to Payment Sent === 'Yes'. I have yet to receive this payment.